scripts/mvftoms.py

```diff
@@ -281,6 +328,77 @@ def _cp_index(a1, a2, pol):
     else:
         print('No calibration products will be applied')
 
+    # select a subset of antennas
+    avail_ants = [a.name for a in dataset.ants]
+    dump_ants = options.antenna if len(options.antenna) != 0 else \
+                avail_ants
+    # some users may specify comma-separated lists although we said the switch should
+    # be specified multiple times. Put in a guard to split comma separated lists as well
+    split_ants = []
+    for a in dump_ants:
+        split_ants += [x.strip() for x in a.split(",")]
+    dump_ants = split_ants
+
+    if not set(dump_ants) <= set(avail_ants):
+        raise ValueError("One or more antennas cannot be found in the dataset. "
+                         "You requested {0:s}, but only {1:s} are available.".format(
+                         ", ".join(["'{}'".format(f) for f in dump_ants]),
+                         ", ".join(["'{}'".format(f) for f in avail_ants])))
+
+    if len(dump_ants) == 0:
+       print('User antenna criterion resulted in empty database, nothing to be done. '
+             'Perhaps you wanted to select from the following: {}'.format(
+             ", ".join(["'{}'".format(f) for f in avail_ants])))
+
+    print('Per user request the following antennae will be dumped: {}'.format(
+        ", ".join(["'{}'".format(f) for f in dump_ants])))
+
+    # select a subset of targets
+    avail_fields = [f.name for f in dataset.catalogue.targets]
+    dump_fields = options.field if len(options.field) != 0 else \
+                  avail_fields
+
+    # some users may specify comma-separated lists although we said the switch should
+    # be specified multiple times. Put in a guard to split comma separated lists as well
+    split_fields = []
+    for f in dump_fields:
+        split_fields += [x.strip() for x in f.split(",")]
+    dump_fields = split_fields
+
+    if not set(dump_fields) <= set(avail_fields):
+        raise ValueError("One or more fields cannot be found in the dataset. "
```

Comment:
I think we use `RuntimeError` for these kinds of errors in the rest of the script.